webook/arrangement/models.py

```diff
@@ -854,21 +856,24 @@ class Meta:
         verbose_name = _("Event")
         verbose_name_plural = _("Events")
 
-    PLANNING_SILO = 'planning_silo'
-    ANALYSIS_SILO = 'analysis_silo'
-    REQUISITIONING_SILO = 'requisitioning_silo'
-    PRODUCTION_SILO = 'production_silo'
-    ARCHIVE_SILO = 'archive_silo'
-
-    SILO_CHOICES = (
-        (PLANNING_SILO, PLANNING_SILO),
-        (REQUISITIONING_SILO, REQUISITIONING_SILO),
-        (ANALYSIS_SILO, ANALYSIS_SILO),
-        (PRODUCTION_SILO, PRODUCTION_SILO),
-        (ARCHIVE_SILO, ARCHIVE_SILO),
+    ARRANGEMENT_EVENT = 'arrangement_event'
+    COLLISION_EVENT = 'collision_event'
+    HOLIDAY_EVENT = 'holiday_event'
+    EVENT_TYPE_CHOICES = (
+        ( ARRANGEMENT_EVENT, ARRANGEMENT_EVENT ),
+        ( HOLIDAY_EVENT, HOLIDAY_EVENT ),
```

Comment:
Is this, and the other ones, suppose to be a choice field in django?

Reply:
Yes. It came a bit prematurely, as my original implementation plan dictated that I needed to store collision events (as opposed to simply not creating them, as the current implementation does). That required EVENT_TYPE so as to discriminate on the calendars get events queries, we wouldn't want to show the collision events to the users anywho unless in very specific circumstances.
I decided to add these extra types, as I knew that eventually they would be needed. There has been a stated wish to be able to create events that do not exist under the umbrella of arrangements - which makes sense. The EVENT_TYPE then would be the ideal place to get this done - hence HOLIDAY_EVENT as a starting point. 
It doesn't really belong here though, that is true - but once it was done and in I felt it might as well stay.